test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSFramework.scala

```diff
@@ -61,10 +71,26 @@ final class ScalaJSFramework(
   }
 
   private def frameworkInfoLauncher = {
+    val prefix = optionalExportsNamespacePrefix
     val name = jsonToString(frameworkName.toJSON)
     val code = s"""
-      new org.scalajs.testinterface.internal.InfoSender($name).initAndSend();
+      new ${prefix}org.scalajs.testinterface.internal.InfoSender($name).initAndSend();
     """
     new MemVirtualJSFile(s"testFrameworkInfo.js").withContent(code)
   }
+
+  private[testadapter] def optionalExportsNamespacePrefix: String = {
+    // !!! DUPLICATE code with ScalaJSPlugin.makeExportsNamespaceExpr
```

Comment:
We'll probably have to do better than this, since this is not likely to go away anytime soon. Also, there is not really a big reason you cannot put this into `tools.javascript` or something, is there?
I'm fine if you leave this as it is for now and add a cleanup issue.